source/agora/node/Ledger.d

```diff
@@ -749,8 +749,7 @@ public class Ledger
 
     ***************************************************************************/
 
-    public string validateBlock (in Block block,
-        string file = __FILE__, size_t line = __LINE__) nothrow @safe
```

Comment:
I think it's not bad to remain this prototype and the previous line calling this function as it was.

Reply:
But recently the logging was removed that used the parameters `file` and `line`.  I think the intention now is to only return info in the `string` returned and do no logging here. @Geod24 ?

Reply:
Also I might need to add a new default param here soon.

Reply:
OK. I understand your intention. My wonder was why the two parameters were removed only on the function, not on the `accecptBlock`.

Reply:
I didn't see that. I should remove from `acceptBlock` also as they are now unused there also.

Reply:
But I think it's ok they are removed only from this function. Do we happen to need to use the parameters later?

Reply:
Updated